### PR TITLE
Add Prettier

### PR DESCRIPTION
**Pull request by erandagan**
Configured Prettier and formatted all JS source files.

The diff on this PR is rather scary but no code changes were made, the code was only reformatted.

### CODE REVIEW

**Comment by Suraj-Tiwari** [comment]
@Trancever add bot for this.

**Comment by Trancever** [comment]
@Suraj-Tiwari What do you mean exactly? What bot?

File: AdChoicesManager.js [hunk 1]
```diff
@@ -1,83 +1,84 @@
 // @flow
-import React, {PropTypes} from 'react';
+import React, { PropTypes } from 'react';
```
Comment (Trancever) [inline AdChoicesManager.js]: Are PropTypes used in this component? It's not related to this PR but we should get rid of it so that's why i am asking.

Comment (erandagan) [reply AdChoicesManager.js]: Unrelated to this PR, but yeah there are some Flow and ESLint errors that should be fixed (unused imports, broken types and such)

**Comment by Suraj-Tiwari** [comment]
I mean github bot to validate Pr

>

**Comment by Trancever** [comment]
I don't think we need bot.. I will setup circleci next week.

**Comment by Suraj-Tiwari** [comment]
Oops... I forgot About CI

**Comment by erandagan** [comment]
So can we merge this?

**Comment by Suraj-Tiwari** [comment]
For next 6 days request all pr check from Trens i don't have laptop with me.

**Pull request merged**